cookbook/core/remote_flyte/customizing_resources.py

```diff
@@ -36,7 +36,7 @@
 from flytekit import Resources, task, workflow
 
 
-@task(requests=Resources(cpu="1", mem="2048"), limits=Resources(cpu="2", mem="4096"))
+@task(requests=Resources(cpu="1", mem="6Mi"), limits=Resources(cpu="2", mem="100Mi"))
```

Comment:
I would set the request to a number that can work... because with memory being an overcommitable resource, it's possible to place this pod on a node that can only provide ~6MB of memory and any increase would result in OOM...
https://kubernetes.io/docs/tasks/administer-cluster/out-of-resource/
https://docs.okd.io/latest/nodes/clusters/nodes-cluster-overcommit.html

Maybe something like this:
```suggestion
@task(requests=Resources(cpu="1", mem="100Mi"), limits=Resources(cpu="2", mem="150Mi"))
```

Reply:
Thanks Haytham. Done